_src/_assets/js/dashboard-charts.js

```diff
@@ -104,10 +105,27 @@
         value: d.positiveIncrease + d.negativeIncrease,
       })),
     ].slice(0, cdcData.length)
-    // debugger
     const cdcChartContainer = d3.select('#cdc-test-chart')
     const ctChartContainer = d3.select('#ct-test-chart')
     const yMax = d3.max(ctData, d => d.value)
+    const cdcSpeciumSum = d3.sum(transformedCdcData, d => {
```

Comment:
slightly more concise way to write this:
`const cdcSpeciumSum = d3.sum(transformedCdcData, d => +d.value)`